README.markdown

```diff
@@ -76,6 +76,11 @@ In place of a version pointer like "0.10", you can use the special default alias
     nvm use stable
     nvm run unstable --version
 
+If you want to install a new version of Node.js and migrate npm packages from a previous version:
+
+    nvm install v0.10.40 --reinstall-packages-from=0.10.39
+    nvm install v0.12.7 --reinstall-packages-from=0.12.6
```

Comment:
This looks awesome. Probably should also add `nvm install iojs --reinstall-packages-from=iojs` right below the node example, so that's covered too, but otherwise LGTM!

Thanks again!